docs/en/observability/index.asciidoc

```diff
@@ -151,3 +151,6 @@ include::fields-reference.asciidoc[leveloffset=+1]
 
 // Tutorials
 include::tutorials.asciidoc[]
+
+// add redirects file
+include::legacy/redirects.asciidoc[]
```

Comment:
The `legacy` directory is specific to apm-server. I don't think it's necessary here.

Reply:
Thanks, removed